app/components/publications/urgency-level-checkbox.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="auk-form-group">
+  {{#if this.urgencyLevels}}
```

Comment:
Je wil hier wachten tot de volledige `loadData` taak afgewerkt is zodat ook `this.checkboxValue` geinitialiseerd is. Je kan hiervoor de derived state van ember-concurrency gebruiken (http://ember-concurrency.com/docs/derived-state/)